yt_dlp/extractor/abc.py

```diff
@@ -53,8 +53,9 @@ class ABCIE(InfoExtractor):
         'info_dict': {
             'id': '6880080',
             'ext': 'mp3',
-            'title': 'NAB lifts interest rates, following Westpac and CBA',
+            'title': 'NAB lifts interest rates, following Westpac and CBA - ABC listen',
```

Comment:
Should be processing the title with `lambda title: re.sub(r'\s+-\s+ABC \w+$', '', title)`?